### PR TITLE
Make badges shine again

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

File: .scrutinizer.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+filter:
+    excluded_paths:
+        - 'Tests/*'
+        - 'vendor/*'
+tools:
+    external_code_coverage: true
+    php_analyzer: true
+    sensiolabs_security_checker: true
+checks:
+    php: true
```
Comment (stof) [inline .scrutinizer.yml]: should be removed. I'm already managing the scrutinizer config inside scrutinizer itself

**Comment by MarioBlazek** [comment]
@stof updated as requested.

**Comment by stof** [comment]
@MarioBlazek I was talking about the whole scrutinizer config actually

**Comment by MarioBlazek** [comment]
@stof done.

**Comment by stof** [comment]
Regarding the Scrutinizer badges, the issue is that they become ``unknown`` when the latest commit in the repo is too old, because Scrutinizer deletes the data after some time, even when it is the most uptodate one.

**Comment by MarioBlazek** [comment]
@stof thanks for info. I think it is best to close this PR.

**Comment by stof** [comment]
Actually, they told me that they keep the latest inspection. But for this package, the latest inspection was before the refactoring of their storage backend, and they removed all old inspection at some point, so dropping it.

**Comment by TomasVotruba** [comment]
What's a blocker here? It seems like to much detailed review just for adding a badge

**Pull request closed without merging**